server/network.js

```diff
@@ -470,9 +470,11 @@ network.listNetwork_ = function(networkPath) {
  */
 network.incrementalEdges_ = function(file, genes, nodes) {
   var oldNodes = {};
-  nodes.forEach(function(node) {
-    oldNodes[node.id] = true;
-  });
+  if (nodes.length) {
```

Comment:
if nodes.length == 0, then nodes.forEach will not enter the inner function. the if condition is thus redundant?